### PR TITLE
Fix signed build break

### PR DESCRIPTION
**Pull request by jaredpar**
Correct the reference to the deleted BuildAndTest.proj file to use the build.ps1 script.

### CODE REVIEW

**Comment by jaredpar** [comment]
CC @jasonmalinowski, @tannergooding

File: src/Tools/MicroBuild/Build.proj [hunk 1]
```diff
@@ -37,7 +37,7 @@
     <MSBuild Projects="$(RepoRoot)src\NuGet\NuGet.proj" />
     <MSBuild Projects="$(RepoRoot)src\Setup\SetupStep2.proj" Properties="$(SetupStep2Properties);DeployExtension=false" />
 
-    <MSBuild Projects="$(RepoRoot)BuildAndTest.proj" Targets="Test" Condition="'$(SkipTest)' != 'true'" Properties="Test32=true" />
+    <Exec Command="powershell -noprofile -executionPolicy ByPass -file $(RepoRoot)build\scripts\build.ps1 -testDesktop -msbuildDir &quot;$(MSBuildBinPath)&quot;" Condition="'$(SkipTest)' != 'true'" />
```
Comment (jasonmalinowski) [inline src/Tools/MicroBuild/Build.proj]: Do we care about quoting if $(RepoRoot) has a space?

Comment (jaredpar) [inline src/Tools/MicroBuild/Build.proj]: In theory yes. But we don't support that yet.

**Comment by jaredpar** [comment]
Kicked off this build to verify https://devdiv.visualstudio.com/DevDiv/_build/index?buildId=818503&_a=summary

**Comment by jaredpar** [comment]
Didn't fix the bug. I tested locally on Debug builds and it works. It fails on Release builds because I missed the right switch. 😦

**Comment by tannergooding** [comment]
Can we fast track this to dev16?

**Comment by jaredpar** [comment]
@tannergooding note taht it didn't fix the problem. I have a nother commit htat should and I'm validating it now in Microbuild.

https://devdiv.visualstudio.com/DevDiv/_build/index?buildId=818560&_a=summary

**Comment by tannergooding** [comment]
Can we fast track both to dev16 once merged :)

**Pull request merged**